__tests__/unit/lib/models/Model.test.ts

```diff
@@ -33,13 +33,13 @@ describe('Model', () => {
       identifier: 'bc8c928b-dafb-4e61-91b3-7c3e5b93a900',
       sourceId: 'd0c22546-6ae6-4ba9-a141-222cc6eea16d',
       targetId: 'cc07d17e-8450-4adf-84d1-ea7d92ec01ab',
-      type: RelationshipType.Aggregation,
+      type: RelationshipType.Aggregation.toLowerCase(),
```

Comment:
Hi, @rodrigolimoes . Can we change the enum values to be in lowercase?

Reply:
Hi, @Diorbert.
Yes, we can change the enum values to be lowercase.
I am changing this in another branch.